frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -467,6 +463,29 @@ const ManageHostsPage = ({
     }
   );
 
+  // migrate users with current local storage based solution to db persistence
+  const locallyHiddenCols = localStorage.getItem("hostHiddenColumns");
+  if (locallyHiddenCols) {
+    console.log("found local hidden columns: ", locallyHiddenCols);
+    console.log("migrating to server persistence...");
+    (async () => {
+      if (!currentUser) {
+        // for type checker
+        return;
+      }
+      const parsed = JSON.parse(locallyHiddenCols) as string[];
```

Comment:
Not necessary to update here but I'm always in favor of wrapping `JSON.parse` in a try/catch, since it can fail and bail on the rest of the script.